README.md

```diff
@@ -49,36 +48,79 @@ const client = new Realtime(ABLY_API_KEY);
 const spaces = new Spaces(client);
 ```
 
-Create a new space instance and have members enter that space:
+### Create or retrieve a space
+
+A `Space` instance can be created, or an existing space retrieved using the `get()` method. You can connect to multiple spaces, but only once per `get()` operation. Spaces are identified by their unique, unicode string name.
+
+You can pass an `offlineTimeout` option when creating a space to specify the length of time in milliseconds after a member loses their connection before that member's object is removed from the space. The default is 120,000ms (2 minutes).
+
+The following is an example of creating a space with the default `offlineTimeout`:
+
+```ts
+const space = spaces.get("exampleSpace");
+```
+
+### Enter or leave a space
+
+A space can be entered using the `enter()` method. The request can contain an optional `profileData` JSON-serializable object. On entering a space any clients subscribed to that space's `membersUpdate` will be notified of the new member and associated `profileData`.
+
+The following is an example of entering a space with some basic profile data:
 
 ```ts
-const space = spaces.get('mySpace');
 space.enter({
-  username: 'Example User',
-  avatar: 'https://example.com/users/1234.png',
+  username: "Example User",
+  avatar: "https://example.com/users/1234.png",
 });
 ```
 
-Subscribe to updates from other members in the space. The `membersUpdate` event will fire when a member's connection status updates, e.g. when a member joins or leaves the space:
+A `Space` can be left using the `leave()` method. On leaving a space any clients subscribed to that space's `membersUpdate` will be notified.
+
+The following is an example of leaving a space:
+
+```ts
+space.leave();
+```
+
+### Realtime space updates
+
+You can subscribe to realtime events by using `on(event, callback)`. Currently the only supported events are `membersUpdate()` and `locationUpdate()`.
+
+#### Subscribe to member updates
+
+Member data can be used to track events relating to a user, for example when a member joins or leaves a space.
+
+To receive updates from members in a space you can subscribe to that space's `membersUpdate`. The `membersUpdate` event will trigger when a member's connection status updates. The response returns an array of the current `SpaceMember` objects within that space.
```

Comment:
the membersUpdate is an overarching event containing all the realtime updates happening in the space. We also have (or will soon have) more scoped events and listeners. Can we please share this context somewhere?